### PR TITLE
Forgot password

### PR DESCRIPTION
**Pull request by Shailendra12219**
Updated code

### CODE REVIEW

File: client/containers/Booking/Booking.jsx [hunk 1]
```diff
@@ -61,44 +58,42 @@ export class Booking extends Component {
                 {
                     this.props.appointment.isSubmitted ?
                         <div className="requestDiv">
-                            <p className="requestText">Your Request has been submitted successfully for Approval</p>
+                            <p className="requestText">You will receive an email once your request is reviewed</p>
```
Comment (hkusanic) [inline client/containers/Booking/Booking.jsx]: Shouldn't this be like this (take a look at the email I've sent previously)
Your request has been submitted successfully for approval.
You will receive an email once your request is reviewed.

File: client/containers/Booking/Booking.jsx [hunk 2]
```diff
@@ -61,44 +58,42 @@ export class Booking extends Component {
...
-                            className="bookingStyle"
-                            frameBorder="0"
-                            allowtransparency="true">
-                        </iframe>
-                        : ''
+                        (
+                            (this.props.appointment.appointmentData && this.props.appointment.appointmentData.Appointment && !this.props.appointment.appointmentData.Appointment.approved &&
+                                !this.props.isSubmitted) ?
+                                < div className="requestDiv">
+                                    <p className="requestText">Your request is under reviewed, you will receive an email once your request is reviewed</p>
```
Comment (hkusanic) [inline client/containers/Booking/Booking.jsx]: Just paste this text:
Your request is under review.
You will receive an email once your request is reviewed.

Comment (Shailendra12219) [reply client/containers/Booking/Booking.jsx]: made the changes, please have a look

File: client/containers/Booking/Booking.jsx [hunk 3]
```diff
@@ -61,44 +58,42 @@ export class Booking extends Component {
                 {
                     this.props.appointment.isSubmitted ?
                         <div className="requestDiv">
-                            <p className="requestText">Your Request has been submitted successfully for Approval</p>
+                                    <p className="requestText">Your request is under reviewed, you will receive an email once your request is reviewed</p>
```
Comment (hkusanic) [inline client/containers/Booking/Booking.jsx]: Your request is under review.
You will receive an email once your request is reviewed.

**Pull request merged**